pages/app/models/refinery/page.rb

```diff
@@ -36,7 +26,7 @@ def self.options
           friendly_id_options[:use] << :scoped
           friendly_id_options.merge!(scope: :parent)
         end
-        friendly_id_options[:use] << :globalize
+        friendly_id_options[:use] << :mobility
```

Comment:
Can you try moving this line up so it comes *before* `:scoped` in the array to `friendly_id_options[:use]` ?

Reply:
I've tried but it sadly does not change anything 😢